arrow-libs/optics/arrow-optics/src/commonTest/kotlin/arrow/optics/EveryTest.kt

```diff
@@ -39,13 +38,13 @@ class EveryTest : StringSpec({
 
       "asFold should behave as valid Fold: combineAll" {
         checkAll(Arb.list(Arb.int())) { ints ->
-          fold(Monoid.int(), ints) shouldBe ints.sum()
+          fold(0, { x, y -> x + y }, ints) shouldBe ints.sum()
         }
       }
 
       "asFold should behave as valid Fold: fold" {
         checkAll(Arb.list(Arb.int())) { ints ->
-          fold(Monoid.int(), ints) shouldBe ints.sum()
+          fold(0, { x, y -> x + y }, ints) shouldBe ints.sum()
```

Comment:
**Nit:** We could use `Int::plus` instead
```suggestion
          fold(0, Int::plus, ints) shouldBe ints.sum()
```

Reply:
We've had bad experiences in the past with the implicit conversion from `A.(A) -> A` into `(A, A) -> A`, so I would rather leave it this way.